<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>665</width>
    <height>514</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Search Employee</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Enter Employee Name</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="employeeNametxt">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>70</y>
     <width>351</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="plainText">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="searchBut">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>160</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="searchResulttxt">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>220</y>
     <width>601</width>
     <height>251</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
